### ISSUE TITLE
Postgres example? Enums do not work with Postgres?

### ISSUE THREAD

**Issue by rex-remind101** [mikro-orm/mikro-orm-examples#1]
I tried using the entities in the MySQL example for Postgres, but they don't work. Using the schema generator, `PublisherType` from https://github.com/B4nan/mikro-orm-examples/blob/master/nest/src/entities/Publisher.ts#L17 makes a JSON column, however, `PublisherType` writes/inserts as a string. This causes the following SQL error -
```
UnhandledPromiseRejectionWarning: error: invalid input syntax for type json
 in query: INSERT INTO "publisher" ("name", "type") VALUES ($1, $2) RETURNING "id"
 with params: ["publisher name","local"]
```

**Comment by B4nan** [comment]
Currently you are required to set the type of enum manually to `string` to be able to use `SchemaGenerator`. I am planning to add better support for enums soon. 

This is how the entity is defined in tests:

```typescript
  @Property({ type: 'string', length: 10 })
  type: PublisherType = PublisherType.LOCAL;
```

**Comment by B4nan** [comment]
Fixed in `3.0.0-alpha.35`. Now when you define the enum via `@Enum` decorator, it will automatically read the enum values and decide the type, numeric enums being the default (so as long as you have numeric enums, you do not have to care about type discovery).

https://github.com/mikro-orm/mikro-orm/blob/dev/docs/defining-entities.md#enums

```typescript
@Enum()
type = PublisherType.LOCAL;
```
